### PR TITLE
Update changelog

### PR DESCRIPTION
**Pull request by miparnisari**
*Issue #, if available:* https://github.com/aws-cloudformation/aws-cfn-lint-visual-studio-code/issues/109

*Description of changes:* update changelog with releases made in 2020

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,78 @@
...
+## 0.12.1 - Jun 12, 2020
+
+Features:
+- Add template preview ([#106](https://github.com/aws-cloudformation/aws-cfn-lint-visual-studio-code/pull/106))
+
+## 0.11.3 - Jun 11, 2020
+
+Features:
+- CloudFormation Template Schema 15.0.0 ([#107](https://github.com/aws-cloudformation/aws-cfn-lint-visual-studio-code/pull/107))
+- E1001 error message re-wording ([#105](https://github.com/aws-cloudformation/aws-cfn-lint-visual-studio-code/pull/105))
```
Comment (PatMyron) [inline CHANGELOG.md:12]:
```suggestion
```

this was just fixing one of the tests

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,78 @@
...
+
+## 0.10.11 - Mar 4, 2020
+
+Features:
+- CloudFormation Template Schema 11.3.0 ([#82](https://github.com/aws-cloudformation/aws-cfn-lint-visual-studio-code/pull/82))
+
+## 0.10.10 - Feb 29, 2020
+
+Features:
+- Visual Studio Code extension manifest fields ([#80](https://github.com/aws-cloudformation/aws-cfn-lint-visual-studio-code/pull/80))
```
Comment (PatMyron) [inline CHANGELOG.md:58]:
```suggestion
```

just some Visual Studio Code marketplace listing metadata

**Comment by miparnisari** [comment]
> are new CloudFormation template schema versions noteworthy enough to maintain every week in the CHANGELOG?

is there absolutely no way we could download these on the fly and not require a user update?

**Comment by PatMyron** [comment]
> > are new CloudFormation template schema versions noteworthy enough to maintain every week in the CHANGELOG?
> 
> is there absolutely no way we could download these on the fly and not require a user update?

[only hosted CloudFormation Template Schema was from 2018](https://github.com/aws-cloudformation/aws-cloudformation-template-schema/issues/32)

[new CloudFormation Template Schemas are generated ~weekly by running a Java application](https://github.com/aws-cloudformation/aws-cfn-lint-visual-studio-code/pull/76)

but the new [Resource Specifications](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/cfn-resource-specification.html) used to generate them are usually malformed so the process includes [manually figuring out patches to fix those deficiencies](https://github.com/aws-cloudformation/aws-cfn-lint-visual-studio-code/pulls?q=%22CloudFormation+Template+Schema%22)

---

I'd like to continue documenting the weekly patching process publicly somewhere because tools that depend on the resource specifications borrow each other's patches, but open to the idea

A URL could always host the latest CloudFormation Template Schema and be updated weekly with the new version. The process of fetching newer CloudFormation Template Schema versions would have to not interfere with the experience though

---

**If the [release process gets streamlined](https://github.com/aws-cloudformation/aws-cfn-lint-visual-studio-code/issues/110), the current process seems easiest though**, so we can feed two birds with one scone by knocking that out

**Pull request closed without merging**